Update block-events.md

VactricaKing opened the pull request:
I have discovered a better way to place the blocks using "set_block" and "set_block_at_pos" it seems almost nobody knows it, so I contribute something that in my opinion will greatly improve how to make a double block.

VactricaKing committed:
Update block-events.md

I have discovered a better way to place the blocks using "set_block" and "set_block_at_pos" it seems almost nobody knows it, so I contribute something that in my opinion will greatly improve how to make a double block.

VactricaKing committed:
Update block-events.md

Edited java to bedrock block states in trapdoor

VactricaKing committed:
Update block-events.md

Changed "minecraft:door" to "minecraft:wooden_door" name

VactricaKing commented:
@SmokeyStack Why is it not accepted yet? Is there a problem with my contribution?

SmokeyStack commented on docs/blocks/block-events.md:
````diff
@@ -283,7 +284,13 @@ Removes the current block and replaces it with the defined block in the same pos
 ```json
 "wiki:place_block": {
   "set_block": {
-      "block_type": "minecraft:grass"
````
Please make this a separate codeblock and subheader. That way we can have a normal `set_block` and one with block states

SmokeyStack commented on docs/blocks/block-events.md:
````diff
@@ -295,10 +302,15 @@ Sets a block at a specified position relative to the block.
 <CodeHeader>minecraft:block > events</CodeHeader>
 
 ```json
-"wiki:generate_stone_above": {
````
Please make this a separate codeblock and subheader. That way we can have a normal set_block and one with block states

VactricaKing committed:
Update block-events.md

Separated "set_block" and "set_block_at_pos" with block state and without block state

VactricaKing commented:
@SmokeyStack Done!
